zingolib/src/wallet/describe.rs

```diff
@@ -302,6 +304,55 @@ impl LightWallet {
             .map(|(_index, sk)| *sk)
             .collect::<Vec<_>>()
     }
+
+    #[allow(clippy::result_unit_err)]
```

Comment:
can we put these functions behind a test flag? i know @zancas removed this test flag but these shouldn't be in production code as they seem incomplete and we should write our production code to handle multiple addresses.

Reply:
sure. but. returning an address oughta be a function of a wallet, not a client